extending-valtimo/plugin/custom-plugin-definition.md

```diff
@@ -82,6 +82,63 @@ class TwitterPluginFactory(
 }
 ```
 
+### Plugin categories
+Plugin categories allow a plugin developer to delegate specific functionality to another plugin using an interface.
```

Comment:
Again, what is a plugin category? Maybe it can be rewritten like so:
Plugin categories denote a commonality between plugins of this category. Plugin categories can only apply to interfaces, so any plugin implementing this will belong to that category. When a plugin implements more than one interface with a category, it belongs to multiple categories. This can be used by backend to create plugins that rely on other plugins to be configured, and can be used by frontend to search for plugins in a specific category.

A plugin implementing...